### PR TITLE
update CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by atalyaalon**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-OMG. OMG. OMG. We are really glad you are joining the team!  We set up a contributor's guide that explains all about [setting up ANYWAY's development environment](http://hasadna.github.io/anyway) - prepare to be amazed. 
+OMG. OMG. OMG. We are really glad you are joining the team!  We set up a contributor's guide that explains all about [setting up ANYWAY's development environment](http://hasadna.github.io/anyway) - prepare to be amazed.
 
 ![Contributors guide](../static/img/anyway.png)
 
 This page contains information about reporting issues as well as some tips and
-guidelines useful to experienced open source contributors. 
+guidelines useful to experienced open source contributors.
 
 ## Topics
```
Comment (elazarg) [inline docs/CONTRIBUTING.md:12]: Two broken links below (parentheses separated by a space)

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -98,43 +120,37 @@ Also, if you add an empty file, named `__init__.py` at tests folder, you'll be a
 
 (e.g.: `python -m unittest tests.test_bounding_box_query`)
 
-
-
 **Pull Requests**
 
-Code review comments may be added to your pull requests, keep an eye out for necessary changes before merge approval. If the maintainer decides that your code is good, it will be merged to ANYWAY's Dev repository.
-On your pull requests, don't forget to reference to the fixed bug / feature by adding `Closes #XXXX` or `Fixes #XXXX` to the body of the requests. Also, shortly summarize the changes made and list any
```
Comment (elazarg) [inline docs/CONTRIBUTING.md:106]: either "reference the" or "add a reference to"

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -98,43 +120,37 @@ Also, if you add an empty file, named `__init__.py` at tests folder, you'll be a
...
+Code review comments may be added to your pull requests, keep an eye out for necessary changes before merge approval. If the maintainer decides that your code is good, it will be merged to ANYWAY's `dev` repository.
+On your pull requests, don't forget to reference to the issue using the Linked issues section of the Pull Request. Also, shortly summarize the changes made and list any
```
Comment (elazarg) [inline docs/CONTRIBUTING.md:126]: same below ("and reference them as well")

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -48,38 +48,60 @@ We generally submit **major changes** via direct email to our [mail center](mail
...
+7. Get a code review approval / reject
+8. After approval, merge your PR
+9. Delete your branch after merge
 
 **Clean Code**
 
-See the following Python and JavaScript format guidelines and stick to them: 
+See the following Python and JavaScript format guidelines and stick to them:
 * [Airbnb JavaScript Guidelines] (https://github.com/airbnb/javascript)
 * [Python Style Guide] (https://www.python.org/dev/peps/pep-0008/)
```
Comment (elazarg) [inline docs/CONTRIBUTING.md:96]: Spurious spaces break the markdown

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (atalyaalon) [reply docs/CONTRIBUTING.md:12]: @elazarg thanks! fixed

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (atalyaalon) [reply docs/CONTRIBUTING.md:106]: it's in deleted section

Comment (atalyaalon) [reply docs/CONTRIBUTING.md:106]: @elazarg thanks, I solved the reference sentences grammar

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -98,43 +120,37 @@ Also, if you add an empty file, named `__init__.py` at tests folder, you'll be a
...
 **Pull Requests**
 
-Code review comments may be added to your pull requests, keep an eye out for necessary changes before merge approval. If the maintainer decides that your code is good, it will be merged to ANYWAY's Dev repository.
-On your pull requests, don't forget to reference to the fixed bug / feature by adding `Closes #XXXX` or `Fixes #XXXX` to the body of the requests. Also, shortly summarize the changes made and list any
-known-issues after the future merge (open new issues for them and reference to them as well).
-
-
+Code review comments may be added to your pull requests, keep an eye out for necessary changes before merge approval. If the maintainer decides that your code is good, it will be merged to ANYWAY's `dev` repository.
+On your pull requests, don't forget to add a reference to the issue using the Linked issues section of the Pull Request. Also, shortly summarize the changes made and list any
+known-issues after the future merge (open new issues for them and add a reference them as well).
```
Comment (elazarg) [inline docs/CONTRIBUTING.md:127]:
```suggestion
known-issues after the future merge (open new issues for them and add a reference to them as well).
```

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -48,38 +48,60 @@ We generally submit **major changes** via direct email to our [mail center](mail
...
+* [Airbnb JavaScript Guidelines](https://github.com/airbnb/javascript)
+* [Python Style Guide](https://www.python.org/dev/peps/pep-0008/)
 
 Universally formatted code is crucial in open source projects and promotes ease of writing, reading and maintenance.
 
-
-
 **Logging**
 
 Do not use `print` for messages in Python code! Instead, use the `logging` class.
```
Comment (elazarg) [inline docs/CONTRIBUTING.md:102]:
```suggestion
Do not use `print` for messages in Python code! Instead, use the `logging` module.
```

File: docs/CONTRIBUTING.md [hunk 7]
```diff
@@ -31,10 +31,10 @@ If you are already digging through the code and our [website](https://www.anyway
 * **Title** `**Issue** One-liner regarding the issue you discovered`
 
 * **Description** Explain why this is a bug, how to reproduce it and on which OS and browser you encounter it. If you can't fix it yourself, try to suggest a fix or any leads you found in the code.
-Add any errors thrown from a non-minified stack trace, if possible. 
+Add any errors thrown from a non-minified stack trace, if possible.
 Attach any screenshots that are relevant.
 
-Once you are done with the basics, don't forget to tag it with `bug` and `1 - Ready` and other relevant tags from the following list: `backend`, `frontend`, `easy`
+Once you are done with the basics, don't forget to tag it with `bug` and `1 - Ready` and other relevant tags from the following list: `backend`, `frontend`, `easy`, `good-first-issue`, `news-flash-report`
```
Comment (isavion) [inline docs/CONTRIBUTING.md:37]: 🥳

File: docs/CONTRIBUTING.md [hunk 8]
```diff
@@ -48,38 +48,60 @@ We generally submit **major changes** via direct email to our [mail center](mail
...
+9. Delete your branch after merge
 
 **Clean Code**
 
-See the following Python and JavaScript format guidelines and stick to them: 
-* [Airbnb JavaScript Guidelines] (https://github.com/airbnb/javascript)
-* [Python Style Guide] (https://www.python.org/dev/peps/pep-0008/)
+See the following Python and JavaScript format guidelines and stick to them:
+* [Airbnb JavaScript Guidelines](https://github.com/airbnb/javascript)
+* [Python Style Guide](https://www.python.org/dev/peps/pep-0008/)
```
Comment (isavion) [inline docs/CONTRIBUTING.md:96]: @atalyaalon what do you think about adding a linter/formatters to the project?

Comment (atalyaalon) [reply docs/CONTRIBUTING.md:96]: @isavion we have pylint - check out our tests under .github/workflows

Comment (atalyaalon) [reply docs/CONTRIBUTING.md:96]: @isavion we have pylint - check out our tests under .github, workflows

**Pull request merged**